CMakeLists.txt

```diff
@@ -52,7 +52,6 @@ find_package(JNI REQUIRED)
 find_package(GTest REQUIRED)
 
 find_package(Python 3.6 COMPONENTS Interpreter Development REQUIRED)
```

Comment:
We require python too, but that should no longer be necessary if we do not need it for anything else.

Reply:
Testing removal of this dependency in https://github.com/vespa-engine/vespa/pull/28997